frontend/src/app/stats-chart/stats-chart.component.ts

```diff
@@ -10,6 +10,7 @@ interface ImageStats {
   tag: string;
   img_name: string;
   generator_version: string;
+  test_version: string
```

Comment:
This now seems to require a new modelling. Or perhaps, just omit it? It's currently not used in the Angular UI and it should be fine to omit it (at least I think it'll be fine).